src/components/PostsList.tsx

```diff
@@ -40,7 +40,7 @@ export const PostsList: React.FC<Props> = ({
                   'is-light': post.id !== selectedPostId,
                 })}
```

Comment:
This logic correctly applies the 'is-light' class when the post is not selected. Ensure that this is the intended behavior, as it affects the button's appearance.